### DISCUSSION TITLE
will design token frames nested in sections still export?

### DISCUSSION THREAD

**Discussion by cs8489** [mikaelvesavuori/figmagic#195] [Q&A] [answered]
I have a robust multi-brand library that requires a more nuanced token structure than most usecases.  This makes my Design Tokens page in figma quite unwieldy, specifically because of how many component-specific tokens I need to create.

The simplest way to organize everything without creating endless parent/children nests of frames is by using Figma's Sections feature.

Will FigMagic be able to identify the fonts, families, line heights, spacing, etc frames that I want to export as tokens when nested inside these sections?

attached is an example of my ideal organization
<img width="1319" alt="Screenshot 2023-11-28 at 4 08 20 PM" src="https://github.com/mikaelvesavuori/figmagic/assets/152334244/cd144ba7-f845-4f0b-a1fe-f35c8b5947ad">

**Comment by mikaelvesavuori** [comment, marked as answer]
Hey, sorry about the late reply (Discussions don't ping me for some reason).

You could easily test this on your end by just running Figmagic on your file. Other than that, Figmagic is very particular in how it requires the structure to be. I can't say for sure it's _not_ going to work, but I wouldn't bet on it.
